### ISSUE TITLE
StereoController AddSteroRig is broken

### ISSUE THREAD

**Issue by bostelk** [googlesamples/cardboard-unity#89]
The function will cache the Eyes array and never invalidate it. I added a simple fix to invalidate the cache after the new Eyes have been created.

This fixes adding the StereoRig at runtime.

``` c#
  /// Helper routine for creation of a stereo rig.  Used by the
  /// custom editor for this class, or to build the rig at runtime.
public void AddStereoRig() {
    if (Eyes.Length > 0) {  // Simplistic test if rig already exists.
	  Debug.Log ("Rig already exists");
      return;
    }
    CreateEye(Cardboard.Eye.Left);
    CreateEye(Cardboard.Eye.Right);
#if !UNITY_EDITOR
	eyes = null;
#endif
    if (Head == null) {
      gameObject.AddComponent<CardboardHead>();
    }
#if !UNITY_5
    if (GetComponent<Camera>().tag == "MainCamera" && GetComponent<SkyboxMesh>() == null) {
      gameObject.AddComponent<SkyboxMesh>();
    }
#endif
}
```

**Comment by smdol** [comment]
Thanks for finding that!

**Comment by smdol** [comment]
Should be fixed in the v0.5.2 release.
